Tests/SonarScanner.MSBuild.PreProcessor.Test/PreProcessorTests.cs

```diff
@@ -644,35 +499,51 @@ private void AssertConfigFileExists(string filePath)
             TestContext.AddResultFile(filePath);
         }
 
-        private void AssertDirectoryContains(string dirPath, params string[] fileNames)
+        private static void AssertDirectoryContains(string dirPath, params string[] fileNames)
         {
             Directory.Exists(dirPath);
-            var actualFileNames = Directory.GetFiles(dirPath).Select(f => Path.GetFileName(f));
+            var actualFileNames = Directory.GetFiles(dirPath).Select(Path.GetFileName);
             actualFileNames.Should().BeEquivalentTo(fileNames);
         }
 
         private static void AssertExpectedLocalSetting(string key, string expectedValue, AnalysisConfig actualConfig)
         {
-            var found = Property.TryGetProperty(key, actualConfig.LocalSettings, out Property actualProperty);
+            var found = Property.TryGetProperty(key, actualConfig.LocalSettings, out var actualProperty);
 
             found.Should().BeTrue("Failed to find the expected local setting: {0}", key);
             actualProperty.Value.Should().Be(expectedValue, "Unexpected property value. Key: {0}", key);
         }
 
         private static void AssertExpectedServerSetting(string key, string expectedValue, AnalysisConfig actualConfig)
         {
-            var found = Property.TryGetProperty(key, actualConfig.ServerSettings, out Property actualProperty);
+            var found = Property.TryGetProperty(key, actualConfig.ServerSettings, out var actualProperty);
 
             found.Should().BeTrue("Failed to find the expected server setting: {0}", key);
             actualProperty.Value.Should().Be(expectedValue, "Unexpected property value. Key: {0}", key);
         }
 
-        private static void AssertDirectoryExists(string path)
-        {
+        private static void AssertDirectoryExists(string path) =>
             Directory.Exists(path).Should().BeTrue("Expected directory does not exist: {0}", path);
-        }
 
-        #endregion Checks
+        private static MockRoslynAnalyzerProvider CreateAnalyzerProviderMock() =>
+            new() { SettingsToReturn = new AnalyzerSettings { RulesetPath = "c:\\xxx.ruleset" } };
+
+        private static MockSonarQubeServer CreateSonarQubeServerMock(bool withDefaultRules = true, string organization = null)
```

Comment:
What about this, to make it shorter? The idea is to use `mock` as a verb. I find it equally readable.
```suggestion
        private static MockSonarQubeServer MockSonarQubeServer(bool withDefaultRules = true, string organization = null)
```

Reply:
It works :)

Reply:
...also `CreateAnalyzerProviderMock` :D